index.html

```diff
@@ -1802,8 +1812,13 @@ <h5>ASCII Case Fold Normalization Step</h5>
         
         <section id="CanonicalFoldNormalizationStep">
 			<h5>Unicode Canonical Case Fold Normalization Step</h5>
+               
+        <div class="req spec" id="matching_ASCIIFoldNormalizationStep">
+            <p class="advisement">Case sensitivity is not recommended for most specifications but, in the case of an exception  where the vocabulary allows non-ASCII characters  and which does not want to be sensitive to case distinctions, the <a href="https://www.w3.org/TR/charmod-norm/#CanonicalFoldNormalizationStep" class="termref">'Unicode canonical case fold'</a> approach SHOULD  be used.</p>
```

Comment:
Should "Unicode canonical case fold" be a relative link (so that we can link to the right place no matter in ED or in /TR)?

Reply:
Thanks for spotting that. Fixed for this and 4 others.